feat: #BOT-4 add chatbot component in portal

flmartineau opened the pull request:
- Added chatbot component in portal. 
- Fetches the chatbot.js script from the chatbot public API (https://github.com/OPEN-ENT-NG/chatbots-connector)

Jira ticket : https://jira.support-ent.fr/browse/BOT-4

jcbe-ode committed:
fix(captation): #WB-1000, allow saving a video when max duration is reached

flmartineau commented on src/ts/directives/business/portal/portal.directive.ts:
```diff
@@ -50,11 +59,17 @@ class Directive implements IDirective<PortalScope,JQLite,IAttributes,IController
+
+
+		if (scope.me?.hasWorkflow('fr.cgi.chatbot.controller.ChatbotController|view')){
+			$.ajax('/chatbot/public/js/chatbot.js', {
```
Le script suivant est chargé : 

https://github.com/OPEN-ENT-NG/chatbots-connector/blob/dev/src/main/resources/public/ts/chatbot-script.ts

JinDouang commented on src/ts/directives/business/portal/portal.directive.ts:
```diff
@@ -50,11 +59,17 @@ class Directive implements IDirective<PortalScope,JQLite,IAttributes,IController
 		}
+
+
+		if (scope.me?.hasWorkflow('fr.cgi.chatbot.controller.ChatbotController|view')){
```
fr.openent (pour ton module)

ccreusat committed:
fix(community): #WB2-976, missing community item in header

ccreusat committed:
fix(header), #WB2-976, community link

Headan committed:
fix: #WB2-315, add condition to keep title

Headan committed:
fix: #WB2-432, add var to disable save recorder after save

Commit:
feat: #BOT-4 add chatbot component in portal

feat: #BOT-4 add chatbot component in portal

feat: #BOT-4 add chatbot component in portal
